bin/src/run/watch.js

```diff
@@ -76,7 +76,10 @@ export default function watch(configFile, configs, command, silent) {
 					break;
 
 				case 'END':
-					if ( !silent && isTTY ) stderr( `\nwaiting for changes...` );
+					if ( !silent && isTTY ) {
+						stderr( `\nfinished at ${new Date()}` );
```

Comment:
The default Date format is rather ...quaint?

```sh
$ node -p "new Date().toString()"
Fri Sep 15 2017 15:13:26 GMT-0700 (PDT)
```

Why does this have to be across multiple lines, anyway? `[YYYY-MM-DD HH:MM:SS] waiting for changes...` seems more than sufficient. [date-time](https://www.npmjs.com/package/date-time) looks like a good library for the formatting.

Reply:
I moved the timestamp to the "waiting" line and reformatted it.